torch/sparse/__init__.py

```diff
@@ -4,10 +4,22 @@
 import torch
 from torch import Tensor
 
+# A workaround to support both TorchScript and MyPy:
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from torch import dtype as DType
+else:
+    DType = int
```

Comment:
@ezyang do you happen to know why this is needed? @pearu checked and found that torchscript doesn't support `torch.types._dtype` in the type annotation. That seems like it's going to give problems in other places.

Reply:
Yes, we're going need to clue in TorchScript about this, unfortunately >:(